scripts/travis/before_install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# Used to perform initial setup for Travis CI builds.
+sudo apt-get update -qq
```

Comment:
Does this script make sense, seeing how having this (or equivalent) line present in `.travis.yml` directly is canonical?

Reply:
I think it's slightly better for consistency with the other scripts, i.e. IMHO it's better not to mix commands inside scripts and outside of them. And in the (admittedly unlikely, but not really impossible) case that we need to do something else later, it would be simpler to add it to this script.

If anything, I don't really see any point in separating `before_install` and `install` steps but, well, this does seem to be canonical, so I kept doing this, even if we could save a script (and a line in `.travis.yml`) by just doing this in `install.sh` instead...